### PR TITLE
Ember CLI Meeting Notes 9 November 2017

### PR DESCRIPTION
**Pull request by hjdivad**

### CODE REVIEW

File: ember-cli/2017-11/november-9.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# ember-cli Meeting Notes - 2017-11-09
+
+## Attendees
+
+## Agenda
```
Comment (hjdivad) [inline ember-cli/2017-11/november-9.md]: Please reply here with agenda items.

Comment (Turbo87) [reply ember-cli/2017-11/november-9.md]: `eslint-plugin-ember` situation

Comment (Turbo87) [reply ember-cli/2017-11/november-9.md]: F2F @ EmberConf

Comment (Turbo87) [reply ember-cli/2017-11/november-9.md]: https://github.com/ember-cli/ember-cli/issues/7421

Comment (rwjblue) [reply ember-cli/2017-11/november-9.md]: * ember-try in ember-cli dependencies https://github.com/ember-cli/ember-cli/issues/7421
* `ember-cli-qunit` -> `ember-qunit` ?

**Comment by locks** [comment]
Closing due to lack of actual notes. Thanks y'all!

**Pull request closed without merging**